README.md

```diff
@@ -33,11 +33,14 @@ WIP on Linux guests.
 ```console
 $ cd arceos-vmm
 $ make disk_img
+$
+$ # Copy guest VM binary image files.
 $ mkdir -p tmp
 $ sudo mount disk.img tmp
-$ # Copy guest VM binary image files.
 $ sudo cp /PATH/TO/YOUR/GUEST/VM/IMAGE tmp/
 $ sudo umount tmp
+$
+$ # Otherwise, set `image_location = "memory"` in CONFIG/FILE, then set kernel_path
```

Comment:
add more descriptions here.

you can create a `doc` directory in the root path of this repo and privade guides about how to boot ArceOS-hypervisor upon rk3588 platform, as well as how to set Linux as its first VM.